projects/agent-framework-javascript/agent-framework-javascript.md

```diff
@@ -0,0 +1,97 @@
+# Project Name
+
+This is still up for debate. Until then, the working title is _Agent Framework JavaScript_.
+
+# Preferred Maturity Level
+
+Considering the fact that the project has been adopted by various companies and governments, and powers a number of production use cases, the _impact_ stage seems appropriate. However, the project does not fulfill the "is commonly used in enterprise production environments".
+
+Therefore, the project is estimated to be in the _growth_ stage, approaching the _impact_ stage.
+
+# Project Description
+
+Aries Framework JavaScript has evolved significantly since its inception as a Hyperledger Aries project. Initially, it heavily relied on Hyperledger standards such as DIDComm, Indy, and AnonCreds. However, with advancements in verifiable credential technology and the emergence of new standards, the framework underwent multiple refactoring and modularization processes to maintain interoperability.
+
+This allowed for the inclusion of non-Hyperledger standards like W3C Verifiable Credentials with Data Integrity Proofs, Presentation Exchange, OpenID4VC, and SD-JWT integration. As industry requirements shifted towards greater modularity, it became apparent that a unified framework may be better.
+
+The future direction for Aries Framework JavaScript involves adopting a compartmentalized approach consisting of single-purpose libraries designed to work together seamlessly, building on what is already out there. This transition will take considerable effort and will, therefore, be gradual.
+
+In order to expand the framework's support for standards beyond the Hyperledger ecosystem, a reassessment of its governance was necessary. The Open Wallet Foundation (OWF) has been considered a potential steward due to its commitment to promoting interoperability without directly developing or maintaining standard protocols.
```

Comment:
What is specifically meant with "a reassessment of its governance was necessary"?

Reply:
So that you know, OWF does not get involved in the project's governance. The maintainers of the project, or its Technical Steering Committee, are driving it. 

The project needs to stay compliant with OWF's policies and rules; see the following Technical Charter sample for more details
* https://docs.google.com/document/d/1Sxnktu3d_xLFSZmAmSFbUp1Y1s1E36YvNDupY2AmigU/edit?pli=1

Reply:
@TimoGlastra, good point. I can see I've worded it poorly now that you point it out. The intention was to express that we (maintainers) feel that a reassessment of where the project is hosted is needed, not so much governance.

How about we replace the sentence with the following?

> The framework has reached a pivotal point in its development, one where its current trajectory of adopting non-standard protocols calls for a reassessment of its hosting ecosystem.

Reply:
Looks good, maybe replace hosting with governing?

So it becomes:
> The framework has reached a pivotal point in its development, one where its current trajectory of adopting non-Aries protocols calls for a reassessment of its governing ecosystem.